stdlib/bytes.gr

```diff
@@ -113,6 +123,8 @@ provide let fromString = (string: String) => {
  * @param bytes: The source byte sequence
  * @returns The string representation of the bytes
  *
+ * @example Bytes.toString(b"\x48\x65\x6c\x6c\x6f\x20\x57\x6f\x72\x6c\x64") == "Hello World"
```

Comment:
Maybe also add the same example but like `Bytes.toString(b"Hello World")` so that it doesn't make people think that the bytes sntax is only usable with hex codes